actions/software.mk

```diff
@@ -27,7 +27,7 @@ include ../../../software/config.mk
 CFLAGS += -std=c99 -I../../../software/include
 DESTDIR ?= /usr
 libs += ../../../software/lib/libsnap.a
-LDLIBS += $(libs) -lpthread
+LDLIBS += $(libs) -lpthread -lm
```

Comment:
That is for all applications. Do we want that or do we want to add that in the specific app Makefile as LDLIBS += -lm?